otx/algorithms/detection/configs/instance_segmentation/efficientnetb2b_maskrcnn/tile_pipeline.py

```diff
@@ -27,7 +27,7 @@
 img_norm_cfg = dict(mean=(103.53, 116.28, 123.675), std=(1.0, 1.0, 1.0), to_rgb=True)
 
 train_pipeline = [
-    dict(type="Resize", img_scale=img_size, keep_ratio=False),
+    dict(type="Resize", img_scale=img_size, keep_ratio=True),
```

Comment:
what is wrong with using `keep_ratio=False`?

Reply:
The aspect ratio of remaining right or bottom tiles will be quite different from 1.0. For example, 50x500 tile will be resized to 512x512, and this might affect the accuracy as the instances in it might look quite different ones in other tiles. So I think "keep_ratio" should be True at least for tiling. What do you think? I might be wrong.